modules/pel/peltool/parse_user_data.py

```diff
@@ -40,25 +40,35 @@ def parse(self) -> str:
         else:
             value = self.parseCustom()
 
+        # Catch if value is None, otherwise python crashes
+        if value == None:
+            value = ("PROBLEM with UNRECOGNIZED value from parser creatorID={} compID={} subType={} version={}"
+                     .format(self.creatorID, hex(self.compID), self.subType, self.version))
+            return json.dumps(value)
         return value
 
     def parseCustom(self) -> str:
         import importlib
         name = (self.creatorID.lower() + "%04X" % self.compID).lower()
         try:
             cls = importlib.import_module("udparsers." + name + "." + name)
-            mv = memoryview(self.data)
-            return cls.parseUDToJson(self.subType, self.version, mv)
+            if self.data:
+                mv = memoryview(self.data)
+                return cls.parseUDToJson(self.subType, self.version, mv)
         except ImportError:
-            mv = memoryview(self.data)
-            return json.dumps(hexdump(mv))
+            print("Failed parseCustom ImportError name={} creatorID={} compID={}"
```

Comment:
this just means there isn't a plugin registered for that UserData section, which we'd see for ever PHYP error, so I'd rather not print anything here.

Reply:
Sounds good ...